shared/panels/contributor-sidepanel/contributor-sidepanel.hooks.tsx

```diff
@@ -1,7 +1,23 @@
+import { SIDE_PANEL_SIZE } from "@/shared/constants/side-panel-size";
 import { useSinglePanelContext } from "@/shared/features/side-panels/side-panel/side-panel";
 import { SidePanelConfig } from "@/shared/features/side-panels/side-panels.types";
-import { ContributorSidepanelData } from "@/shared/panels/contributor-sidepanel/contributor-sidepanel.types";
+import {
+  ContributorSidepanelData,
+  ContributorSidepanelType,
+} from "@/shared/panels/contributor-sidepanel/contributor-sidepanel.types";
 
-export function useContributorSidePanel(config?: SidePanelConfig) {
-  return useSinglePanelContext<ContributorSidepanelData>("contributor-detail", config);
+interface useContributorSidePanelConfig extends SidePanelConfig {
+  type?: ContributorSidepanelType;
+}
+export function useContributorSidePanel(config?: useContributorSidePanelConfig) {
+  const { type } = config ?? { type: "extended" };
+  const { open, ...rest } = useSinglePanelContext<ContributorSidepanelData>("contributor-detail", {
+    width: type === "short" ? SIDE_PANEL_SIZE.m : SIDE_PANEL_SIZE.xl,
```

Comment:
I don't think `short` should be used for the type, as we don't want to link the type to the size, no ?